src/module.ts

```diff
@@ -45,7 +45,18 @@ export default defineNuxtModule<PrismicModuleOptions>({
 	}),
 	hooks: {},
 	setup (options, nuxt) {
-		if (!options.endpoint) {
+		// Expose options through public runtime config
+		nuxt.options.runtimeConfig.public ||= {} as typeof nuxt.options.runtimeConfig.public
+		const moduleOptions: PrismicModuleOptions = defu(nuxt.options.runtimeConfig.public.prismic, options)
+		nuxt.options.runtimeConfig.public.prismic = moduleOptions
+
+		nuxt.options.alias['#prismicOptions'] = addTemplate({
+			filename: 'prismicOptions.mjs',
+			write: true,
+			getContents: () => `export default ${JSON.stringify(moduleOptions, undefined, 2)}`
+		}).dst || ''
```

Comment:
Is there a benefit of proxying module options through this virtual alias? I feel like just relying on the `useRuntimeConfig()` composable was more straightforward 🤔

Reply:
Hi @lihbr ! 
First, thanks for your review :) 

Yes, both ways work as well. 
But I created a new template to put only the module's configuration, to have direct access and avoid this syntax.

```ts
const config = useRuntimeConfig()

const prismicPlugin = createPrismic({
  ...config.public.primisc, // <-- THIS SYNTAX
  client,
  linkResolver,
  htmlSerializer,
  ...etc
})
```

For me it's safer and good practice. 

And the end user of the module can access the configuration throughout the application through this alias as well.

Reply:
OK, sounds good to me